go-controller/pkg/ovn/policy.go

```diff
@@ -3,12 +3,8 @@ package ovn
 import (
 	"fmt"
 	"net"
-<<<<<<< HEAD
```

Comment:
you are fixing the issues of the previous revert in this revert. Got to clean this up.

Reply:
^